### PR TITLE
Add a PR template

### PR DESCRIPTION
**Pull request by mavam**
This pull request adds a template for pull requests. The idea is that to signal explicitly to the reviewer how the PR should be approached.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+##  :notebook_with_decorative_cover: Description
+
+:pencil:
+
+## :100: Review Instructions
+
+Please review this pull request as follows:
+
+- [ ] Commit by commit
+- [ ] File by file
```
Comment (rolandpeelen) [inline .github/PULL_REQUEST_TEMPLATE.md:10]:
```suggestion
- [ ] Commit-by-commit
- [ ] File-by-file
```

I think like with 'door-to-door' it needs a `-`

**Review by rolandpeelen — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+##  :notebook_with_decorative_cover: Description
+
+:pencil2:
+
+## :dart: Review Instructions
+
+Please review this pull request as follows:
+
+- [ ] Commit-by-commit
+- [ ] File-by-file
```
Comment (dominiklohmann) [inline .github/PULL_REQUEST_TEMPLATE.md:10]:
```suggestion
##  :notebook_with_decorative_cover: Description

<!-- Describe the change you've made in this section -->

##  :memo: Checklist

- [ ] All user-facing changes have changelog entries.
- [ ] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast).
- [ ] The PR description contains instructions for the reviewer, if necessary.

## :dart: Review Instructions

<!-- Provide instructions for the reviewer here, e.g., review this pull request commit-by-commit, or file-by-file. -->
```

**Review by dominiklohmann — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (0ortmann) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: I like the checkbox thing better (last section). It is easier when the "instructions" are always formatted the same way, i.e., a checkbox. If you want to put clear text you can still add that, right?

Comment (dominiklohmann) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: The reason why I removed the checkboxes from that section is that the PR list view shows remaining tasks, and having a checklist there would mean that we always have one outstanding task for every PR.

Comment (mavam) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: Good point, unfortunately that prevents the checkboxes. We could just a add bullet list that users adapt to their needs, e.g., 

```markdown
<!-- Provide review instructions here. -->

- Commit-by-commit
- File-by-file
```

Comment (dominiklohmann) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: I'd be fine with either.

**Review by dominiklohmann — APPROVED** [review]

**Review by tobim — APPROVED** [review]

**Review by 0ortmann — APPROVED** [review]

**Pull request merged**